src/content/docs/query-your-data/explore-query-data/dashboards/dashboards-api.mdx

```diff
@@ -106,4 +106,4 @@ All dashboard mutations offer a way to ask for errors when being executed. This
   </Collapser>
 </CollapserGroup>
 
-Keep in mind that these are expected errors that we are aware of in advance. You should also check for unexpected errors that will be returned in the standard [GraphQL errors field](https://graphql.org/learn/serving-over-http/#response).
```

Comment:
I don't know why this is recognized as changed. I edited the file directly in github.

Reply:
I'd say we can ignore it since the content doesn't seem to have changed.